src/include/miopen/hip_build_utils.hpp

```diff
@@ -32,6 +32,25 @@
 #include <boost/optional.hpp>
 #include <string>
 
+#ifndef HIP_PACKAGE_VERSION_MAJOR
```

Comment:
Please include config.h

Reply:
fixed

Reply:
@asroy Has this been addressed?
@atamazov You review is still blocking this PR. Are there additional changes needed?